_posts/en/newsletters/2025-12-05-newsletter.md

```diff
@@ -0,0 +1,149 @@
+_New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates._
+
+FIXME:Gustavojfe
+
+- [ldk-node v0.7.0][] is a minor release of this library for building lightning
+  applications that adds support for [splicing][topic splicing] (experimental),
+  serving and paying static invoices for [async payments][topic async payments],
+  as well as many other features, bug fixes, and API improvements.
+
+## Notable code and documentation changes
+
+_Notable recent changes in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo],
+[Lightning BLIPs][blips repo], [Bitcoin Inquisition][bitcoin inquisition
+repo], and [BINANAs][binana repo]._
+
+- [Core Lightning #8728][] fixes a bug that caused `hsmd` to crash when a user
+  entered the wrong passphrase; it now properly handles this user error case and
+  exits cleanly.
+
+- [Core Lightning #8702][] adds a `lightningd-downgrade` tool that downgrades
+  the database version from 25.12 to the previous 25.09 in case of an upgrade
+  error.
+
+- [Core Lightning #8735][] fixes a long-standing bug where some on-chain spends
+  could disappear from CLN’s view during a restart. Upon startup, CLN rolls back
+  the latest 15 blocks (by default), resets the spend height of UTXOs spent in
+  those blocks to `null` and then rescans. Previously, CLN failed to rewatch
+  those UTXOs, which could cause CLN to keep relaying [channel
+  announcements][topic channel announcements] that had already been closed, or
+  to miss important on-chain spends. This PR ensures that these UTXOs are
+  rewatched during startup and adds a one-time backward scan to recover any
+  spends that were previously missed due to this bug.
+
+- [LDK #4226][] begins validating the amount and CLTV fields of received
+  [trampoline][topic trampoline payments] onions against the outer onion. It
+  also adds three new local failure reasons:
+  `TemporaryTrampolineFailure`,`TrampolineFeeOrExpiryInsufficient`, and
+  `UnknownNextTrampoline` as a first step towards supporting trampoline payment
+  forwarding.
+
+- [LND #10341][] fixes a bug where the same [Tor][topic anonymity networks]
+  onion address was duplicated in the node announcement and in the `getinfo`
+  output whenever the hidden service was restarted. The PR ensures the
+  `createNewHiddenService` function never duplicates an address.
+
+- [BTCPay Server #6986][] introduces `Monetization`, which allows server admins
+  to require a `Subscription` (see [Newsletter #379][news379 btcpay]) for user
+  login, enabling ambassadors to monetize their work. There’s a default
```

Comment:
Might be good to define ambassadors here and perhaps a bigger picture of the goal here as it wasnt immediately clear to me what this feature was for @Gustavojfe